frontend/components/messenger/message_list_item.jsx

```diff
@@ -38,13 +38,13 @@ class MessageListItem extends React.Component {
   }
 
   render() {
-    const { body, created_at, id } = this.props.message;
+    const { body, created_at, id, user } = this.props.message;
 
     let username = '';
     let img_url = '';
-    if (this.props.user) {
-      username = this.props.user.username;
-      img_url = this.props.user.img_url;
+    if (user) {
```

Comment:
in the `showDeleteButton()` function you are missing a space between closing paren and opening curly